src/compiler/python_generator.cc

```diff
@@ -43,26 +43,12 @@
 #include <tuple>
 #include <vector>
 
+#include "src/compiler/config.h"
 #include "src/compiler/generator_helpers.h"
 #include "src/compiler/python_generator.h"
-#include <google/protobuf/io/coded_stream.h>
-#include <google/protobuf/io/printer.h>
-#include <google/protobuf/io/zero_copy_stream.h>
-#include <google/protobuf/io/zero_copy_stream_impl_lite.h>
-#include <google/protobuf/descriptor.pb.h>
-#include <google/protobuf/descriptor.h>
 
 using grpc_generator::StringReplace;
 using grpc_generator::StripProto;
-using google::protobuf::Descriptor;
-using google::protobuf::FileDescriptor;
-using google::protobuf::MethodDescriptor;
-using google::protobuf::ServiceDescriptor;
-using google::protobuf::compiler::GeneratorContext;
-using google::protobuf::io::CodedOutputStream;
-using google::protobuf::io::Printer;
-using google::protobuf::io::StringOutputStream;
-using google::protobuf::io::ZeroCopyOutputStream;
```

Comment:
So unfortunately, that wasn't quite what I had in mind :-)

If you look at the config file, it typedefs everything into a grpc::protobuf namespace, so you can simply replace all the `using google::protobuf::xxx` with `using grpc::protobuf::xxx`, and not change the rest of the code.